dev-lang/ldc2/ldc2-1.36.0-r2.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit flag-o-matic multilib-build cmake
+
+MY_PV="${PV//_/-}"
+MY_P="ldc-${MY_PV}-src"
+SRC_URI="https://github.com/ldc-developers/ldc/releases/download/v${MY_PV}/${MY_P}.tar.gz"
+S=${WORKDIR}/${MY_P}
+
+DESCRIPTION="LLVM D Compiler"
+HOMEPAGE="https://github.com/ldc-developers/ldc"
+KEYWORDS="~amd64 ~arm64 ~x86"
+LICENSE="BSD"
+# For first bump, set the subslot below to 0 and only increase if there
+# is an actual ABI bkreakage.
+SLOT="$(ver_cut 1-2)/$(ver_cut 3)"
+
+IUSE="static-libs"
+
+# Upstream supports LLVM 11.0 through 17.0.
+LLVM_COMPAT=( {15..17} )
+DLANG_COMPAT=( dmd-2_{106..107} gdc-13 ldc2-1_{35..36} )
+
+inherit llvm-r1 dlang-single
+
+REQUIRED_USE=${DLANG_REQUIRED_USE}
+DEPEND="
+	${DLANG_DEPS}
+	$(llvm_gen_dep '
+	  sys-devel/llvm:${LLVM_SLOT}=
+	')
+"
+IDEPEND=">=app-eselect/eselect-dlang-20140709"
+RDEPEND="
+	${DEPEND}
+	${IDEPEND}
+"
+BDEPEND=${DLANG_DEPS}
```

Comment:
Missing quotes on this and REQUIRED_USE.

Reply:
I think that you are commenting on the style choice and not the fact that those vars are unquoted is an bug, as in there won't be any globbing or something else equally bad. I have no strong preference on the style if that's the case, I've mostly tried to keep the same as was used in the eclasses which is the one from python-r1.eclass. If you want this change I expect that it should be applied to all ebuilds, correct (but leave the eclasses unchanged)?